rotkehlchen/chain/ethereum/utils.py

```diff
@@ -48,6 +48,10 @@ def token_normalized_value_decimals(token_amount: int, token_decimals: int | Non
     return token_amount / (FVal(10) ** FVal(token_decimals))
 
 
+def normalized_fval_value_decimals(amount: FVal, decimals: int) -> FVal:
```

Comment:
Don"t reinvent the wheel and make same functions. 

There is already one doing this `token_normalized_value_decimals`

Reply:
we discussed this will go in another PR